orbit/pkg/luks/luks_linux.go

```diff
@@ -0,0 +1,221 @@
+//go:build linux
+
+package luks_runner
+
+import (
+	"context"
+	"crypto/rand"
+	"errors"
+	"fmt"
+	"math/big"
+	"regexp"
+	"time"
+
+	"github.com/fleetdm/fleet/v4/orbit/pkg/dialog"
+	"github.com/fleetdm/fleet/v4/orbit/pkg/lvm"
+	"github.com/fleetdm/fleet/v4/server/fleet"
+	"github.com/rs/zerolog/log"
+	"github.com/siderolabs/go-blockdevice/v2/encryption"
+	"github.com/siderolabs/go-blockdevice/v2/encryption/luks"
```

Comment:
a divergence from the POC package used as I ran into cross compilation issues with go-cryptsetup.  This package still only compiles on linux, thus the use of `luks_stub.go`